README.md

```diff
@@ -467,34 +470,46 @@ notification intent that the Klaviyo SDK requires in order to track opens when y
 
 ## In App Forms
 
+In-app forms are messages displayed to mobile app users while they are actively using your app. 
+You can create new in-app forms in a drag-and-drop editor in the Sign-Up Forms tab in Klaviyo.
+
+[//]: # (TODO -- add help doc link)
+
 ### Prerequisites
-- Using version 3.2.0 or higher
-- You need to have Klaviyo `analytics` imported and the company ID must be set
+- Version 3.2.0 or higher
+- Klaviyo `analytics` and `forms` packages
 
 ### Setup
-`registerForInAppForms` can be chained with your initialization code to allow for form display.
+Once you've created an in-app form in your Klaviyo account, to display it add the following code to your application. 
+
 ```kotlin
+import com.klaviyo.analytics.Klaviyo
+import com.klaviyo.forms.registerForInAppForms
+
+// You can register as soon as you've initialized
 Klaviyo
     .initialize("KLAVIYO_PUBLIC_API_KEY", applicationContext)
     .registerForInAppForms()
-```
-You can also call it publicly anywhere you can access the Klaviyo forms module:
-```kotlin
-Klaviyo.registerForInAppForms() // make sure you only call this after initializing Klaviyo company ID
+
+// ... or any time thereafter
+Klaviyo.registerForInAppForms()
 ```
 
 ### Behavior
 
-Once you call the `registerForInAppForms` function, the web view will persist unattached to UI until a form is ready to be shown.
-This allows you to choose *when* forms will start to overlay on your App UI. It also means that wherever you decide to call this will impact the behavior.
-Consider how often you want to register for forms when you call this, for example:
+Once `registerForInAppForms()` is called, the SDK will load form data for your account and display no more than one form 
```

Comment:
Actually ya I do like your summary better now that I read it out. It gives enough detail without going into specifics behind the scenes